source/Patches/CustomRPC.cs

```diff
@@ -99,6 +99,7 @@ public enum CustomRPC
         SyncCustomSettings,
         FixAnimation,
 
-        AddMayorVoteBank
+        AddMayorVoteBank,
+        UpdateGamePlayerControlData
```

Comment:
You can remove this now.

Reply:
Forgot that line ;) thanks, will remove it tomorrow.